src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -555,6 +555,8 @@ public virtual bool IsAnonymousType
         /// </summary>
         internal virtual bool IsNativeIntegerWrapperType => false;
 
+        public virtual bool IsExtension => false;
```

Comment:
>public virtual bool IsExtension => false; [](http://example.com/codeflow?start=8&length=41)

I think we prefer APIs like that to be abstract

Reply:
Given that its implementation is just a type kind check. Why not make this method non-virtual.